src/components/MovieListPage/MovieListPage.js

```diff
@@ -34,14 +34,16 @@ export default function MovieListPage({
 const ButtonWrapper = styled.div`
   text-align: center;
   margin-top: 35px;
+
+  &.margin {
+    margin-bottom: 110px;
+  }
```

Comment:
```suggestion
    margin-bottom: 110px;
```

Reply:
I guess the liveserver wasn't working... I tried that but it didn't show. Haha! Restarted the server now and it works just fine. Thanks! 😅